package.json

```diff
@@ -39,10 +40,7 @@
     "@babel/polyfill": "^7.0.0",
     "@babel/runtime": "^7.0.0",
     "cbq": "git+https://github.com/bitfinexcom/cbq-js.git",
-    "eosjs": "^20.0.0",
-    "isomorphic-ws": "^4.0.1",
-    "mandelbrot": "github:bitfinexcom/mandelbrot#master",
-    "ws": "^6.1.0"
```

Comment:
hmm, wait a second, `ws-testhelper.js` depends on this, it just keeps working because another dep is installing that indirectly